page for each teammate to customize

nanselmo opened the pull request:
Added a section for each teammate and updated the codeu summer site with instructions.

nanselmo committed:
page for each teammate to customize

KevinWorkman commented on src/main/webapp/aboutus.html:
```diff
@@ -0,0 +1,56 @@
+    <title>About Us</title>
+    <link rel="stylesheet" href="/css/main.css">
+  </head>
+  <body onload="addLoginOrLogoutLinkToNavigation();">
```
I'm pretty sure this won't work since the JavaScript file isn't included. You could include the JavaScript file if you wanted, but I'd lean towards just removing this callback to simplify this page.

KevinWorkman reviewed (approved):
One minor comment. Other than that LGTM!

KevinWorkman reviewed (commented):
Can you also link to this page from `index.html` and possibly `user-page.html`?

nanselmo committed:
removed navBarLoader function call

nanselmo committed:
added navigation loader back in

nanselmo committed:
added link to aboutus.html

nanselmo committed:
added link to aboutus.html in nav bar

nanselmo commented:
I added a link from aboutus.html to the index.html. In order to add a link to the user-page.html, I needed to include the addLoginOrLogoutLinkToNavigation() js method. 

I also added a link to aboutus.html from index.html and the user-page.

KevinWorkman commented:
SGTM. I think it would be okay for the aboutus page to not link to the user page, just to keep it simple? But I also see value in including it, so either way LGTM.

nanselmo committed:
removed JS that links to user page.

nanselmo commented:
Whoops, yup. Sorry I misread your comment's intent on which page to link from/too. Took the link to the user-page out of AboutUs.html

KevinWorkman commented on src/main/webapp/index.html:
```diff
@@ -25,7 +25,7 @@
   <body onload="addLoginOrLogoutLinkToNavigation();">
     <nav>
       <ul id="navigation">
-        <li><a href="/">Home</a></li>
```
I would maaaybe keep the link to home on this page as well, just so the navigation is somewhat consistent? I don't feel super strongly though.

KevinWorkman commented on src/main/webapp/aboutus.html:
```diff
@@ -0,0 +1,56 @@
+  </head>
+    <nav>
+      <ul id="navigation">
+        <li><a href="/">Home</a></li>
```
I would maybe also include a link to `/aboutus.html` here, so the navigation is consistent?

nanselmo committed:
added link to aboutus in nav bar

nanselmo committed:
added link to home in navbar
